### PR TITLE
IOS OSPF Interfaces resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

IOS OSPF Interfaces resource module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

ios_ospf_interfaces
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/module_utils/network/ios/rm_templates/ospf_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+            command = "ip ospf cost {interface_cost}".format(
+                **config_data["cost"]
+            )
+        elif config_data.get("afi") == "ipv6":
+            command = "ipv6 ospf cost"
+            if "interface_cost" in config_data["cost"]:
+                command = "ipv6 ospf cost {interface_cost}".format(
+                    **config_data["cost"]
+                )
+            # if 'dynamic' in config_data['cost'] and config_data['cost']['dynamic']:
```
Comment (rohitthakur2590) [inline plugins/module_utils/network/ios/rm_templates/ospf_interfaces.py:72]: commented code to be removed ?

File: plugins/module_utils/network/ios/rm_templates/ospf_interfaces.py [hunk 2]
```diff
@@ -0,0 +1,921 @@
...
+        #                 "{{ afi }}": {
+        #                     "afi": "{{ afi }}",
+        #                     "cost": {
+        #                         "dynamic": "{{ True if dynamic is defined }}",
+        #                     },
+        #                 },
+        #             },
+        #         },
+        #     },
+        # },
```
Comment (rohitthakur2590) [inline plugins/module_utils/network/ios/rm_templates/ospf_interfaces.py:422]: same as above

**Review by rohitthakur2590 — APPROVED** [review]

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**